packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -139,6 +139,10 @@ List<Widget> _prettyRows(BuildContext context, List<Row> rows) {
   ];
 }
 
+List<Widget> prettyRows(BuildContext context, List<Row> rows) {
+  return <Widget>[..._prettyRows(context, rows)];
```

Comment:
Why are we expanding a list into a new list here instead of just returning the list?

Reply:
Yeah... returning the list directly is a cleaner way. But, I'm going make the function `_prettyRows` public now